x/ccv/provider/keeper/relay.go

```diff
@@ -214,11 +215,17 @@ func (k Keeper) SendVSCPacketsToChain(ctx sdk.Context, chainID, channelID string
 }
 
 // QueueVSCPackets queues latest validator updates for every registered consumer chain
+// failing to GetLastValidators will cause a panic in EndBlock
```

Comment:
Should `GetLastValidators` be renamed here? Is it used inside `QueueVSCPacket`s?

Reply:
Replaced by GetLastBondedValidators whic is called in QueueVSCPackets.